bin/helpers/utils.js

```diff
@@ -115,6 +115,16 @@ exports.setParallels = (bsConfig, args) => {
   }
 };
 
+exports.defaultAuthHash = (bsConfig, args) => {
```

Comment:
Hi there! What do you think about naming this as `setDefaultAuthValue`? So, this method uses the same format as others (action + noun), and also it uses a less agnostic type of data in the name?

Reply:
Renamed it to setDefaultAuthHash